src/main/java/org/cactoos/func/Retry.java

```diff
@@ -92,6 +124,9 @@ public Y apply(final X input) throws Exception {
             } catch (final Exception ex) {
                 error = ex;
             }
+            if (!this.wait.isZero() && !this.wait.isNegative()) {
+                Thread.sleep(this.wait.toMillis());
```

Comment:
@longstone according to the code above, if the call is interrupted, we are expecting an exception to be thrown, so I think you should catch the `InterruptedException`, set `error` to it and then break in the same way it is done above

Reply:
@longstone also we need a test for the interrupt behaviour, so please add a `@todo` for it

Reply:
@victornoel I'm not sure how to do this. Just adding a comment like `@todo #909:30min add a test to test the interrupt behaviour` to the code is sufficient?

Reply:
@longstone you can see other example of todos in the code but you basically have the right idea yes! The message may need to be a bit longer with some context to be sure the person who is assigned the job understand correctly what needs to be done It can be added to the RetryTest class for example, as a comment on the class next to the other `@xxx` comments.

Reply:
@longstone ping? :)